frontend/src/general/userForm/FormFailedDialog.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Dialog from "../dialog/Dialog";
+
+export default class FormFailedDialog extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.dialogElement = React.createRef();
+    this.openDialog = this.openDialog.bind(this);
+  }
+
+  openDialog() {
+    return this.dialogElement.current.openDialog();
+  }
+
+  render() {
+    const { title, errorMessage } = this.props;
+
+    return (
+      <Dialog ref={this.dialogElement} title={title} isError>
```

Comment:
If this is the `<FormFailedDialog/>`, shouldn't the `title` always be the same?

Reply:
Oh wait, I know why I didn't hardcode the title, it's because I'm going to use FormFailedDialog for also Login (i.e. Failed to login) and not just Create User (i.e. Failed to create user), which is why I had it as a prop.